boskos/Makefile

```diff
@@ -74,7 +75,7 @@ service:
 	kubectl apply -f service.yaml
 
 update-config: get-cluster-credentials
-	kubectl create configmap resources --from-file=config=resources.json --dry-run -o yaml | kubectl --namespace="$(NAMESPACE)" replace configmap resources -f -
+	kubectl create configmap resources --from-file=config=resources.yaml --dry-run -o yaml | kubectl --namespace="$(NAMESPACE)" replace configmap resources -f -
```

Comment:
(IMO this will probably break our current deployment)

Reply:
What do you propose we do ?

Reply:
keep the make target with the old config.json, and make a separate PR that upgrades everything? so it will be easier to revert.

Reply:
I think let's split the make target change in a follow up PR